dist/src/main/java/io/zeebe/EnvironmentHelper.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe;
+
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.core.env.Environment;
+
+public class EnvironmentHelper {
+
+  public static boolean isProductionEnvironment(Environment springEnvironment) {
+    boolean result = true;
+
+    if (springEnvironment == null) {
```

Comment:
Is it possible for this to be `null`? What does that imply, i.e. the auto-wiring failed?

Reply:
Yes it's possible for this to be null. 

This happens when auto-wiring was never triggered. The most common situation is when you instantiate a class in a test. Then you could either mock the injected objects, trigger injection somehow, or simply live with them being null.